src/components/TextList.js

```diff
@@ -29,7 +43,8 @@ export class TextList extends React.PureComponent {
             <ActivityIndicator color={colors.accent} style={{ margin: normalize(14) }} />
           )
         }
-        onEndReached={() => this.setState({ listEndReached: true })}
+        onEndReachedThreshold={0.5}
```

Comment:
what is this 0.5 ?

Reply:
Are you asking for `onEndReachedThreshold` or what the value 0.5 achieves?
Have you checked the official React Native documentation? Both is described there: https://reactnative.dev/docs/flatlist#onendreachedthreshold
Maybe in this StackOverflow it is more detailed: https://stackoverflow.com/questions/39366356/what-onendreachedthreshold-really-means-in-react-native